sensu-plugins-gitlab.gemspec

```diff
@@ -24,12 +24,13 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 2.0.0'
+  s.required_ruby_version  = '>= 2.1.0'
   s.summary                = 'Sensu plugins for skel'
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsGitlab::Version::VER_STRING
 
   s.add_runtime_dependency 'gitlab', '~> 4.3'
+  s.add_runtime_dependency 'rest-client', '~> 2.0'
```

Comment:
@majormoses I think this comment for https://github.com/sensu-plugins/sensu-plugins-gitlab/pull/1/files#diff-bfcca617718ef80549f7e07b60028350R27

Reply:
maybe the diff link is messed up but I don't see what you are talking about. There are 2 breaking changes here:
- dropping ruby `< 2.2`
- adding requirement of `rest-client` that has a dependency on a compiler. As previously people did not have to have a compiler to install this it constitutes a breaking change. If you want to avoid this you can use `net/http` instead of `rest-client`

It should look something like this:
```
### Breaking Changes
- dropped ruby 2.1 support (@yuri-zubov sponsored by Actility, https://www.actility.com)
- added `rest-client` as a dependency which requires you to have a local c compiler present to install this plugin (@yuri-zubov sponsored by Actility, https://www.actility.com)

### Added
- Added health-check for gitlab (@yuri-zubov sponsored by Actility, https://www.actility.com)
￼
```